node/routes.py

```diff
@@ -97,10 +95,10 @@ def post_dispute() -> Response:
             "signature": utils.eth_sign(f'{zconfig.SEQUENCER["id"]}{timestamp}'),
         }
         return success_response(data=data)
-
-    zdb.init_txs(req_data["app_name"], req_data["txs"])
+    if req_data['app_name'] != '':
```

Comment:
if this `if` is related to heartbeat txs, it should be removed.

Reply:
It has been removed in the last issue in switching branch.